src/main/java/com/paskar/email/application/service/EmailRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.paskar.email.application.service;
+
+import com.paskar.email.application.mapper.EmailMapper;
+import com.paskar.email.application.model.Email;
+import com.paskar.email.application.repositiory.EmailRepositoryJdbc;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class EmailRepositoryJdbcImpl implements EmailRepositoryJdbc {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public EmailRepositoryJdbcImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public void save(Email email) {
+        String sql = "INSERT INTO first_homework_with_jdbc (recipient,subject, body, date) VALUES (?,?,?,?)";
+        jdbcTemplate.update(sql, email.getRecipient(), email.getSubject(), email.getBody(), email.getDate());
```

Comment:
I am glad that you aware of JdbcTemplate, but it is not that was expected.
I expect to see how you work with Pure JDBC Api (PreparedStatement, ResultSet, Connection, ...).

I will remind you task description:
> Create 2 Pull Request. First one with Pure JDBC API. Second one with Hibernate

Reply:
I was hoping it wouldn't be necessary to work with so "pure" JDBC :)
Okay, I'll fix it, make a new one PR with pure JDBC and send it to you tomorrow